ui-components/src/blocks/ViewItem.scss

```diff
@@ -37,7 +37,7 @@
   @apply tracking-wide;
   @apply font-semibold;
   @apply block;
-  @apply pt-2;
+  padding-top: 10px;
```

Comment:
FYI, `ViewItem` is used on the public app as well, so I'd double-check if this change makes sense there.

Reply:
Yup already checked, all is well